components/textFields/AdvancedOpenTextField.js

```diff
@@ -195,13 +210,20 @@ const AdvancedOpenTextField = (props) => {
     }
 
     const ErrorMsg = () => {
-        if(isInvalid) {
+        if(invalidLength) {
```

Comment:
I wonder if instead of choosing one of the invalid states if (both are true), should we display all invalidity of the VIN?